templates/web/common/site_layout.tt.html

```diff
@@ -216,6 +217,7 @@
 					<ul class="search_and_links">
 						<li><a href="[% lang('menu_discover_link') %]" class="top-bar-links">[% lang('menu_discover') %]</a></li>
 						<li><a href="[% lang('menu_contribute_link') %]" class="top-bar-links">[% lang('menu_contribute') %]</a></li>
+						<li><a href="[% lang('footer_producers_link') %]" class="top-bar-links">[% lang('footer_producers') %]</a></li>
```

Comment:
I don't think we have room for this link at all resolutions / all languages. Maybe we could display it only on very large screens.

![image](https://github.com/openfoodfacts/openfoodfacts-server/assets/8158668/2a73855a-97b7-44f2-b485-1fbc51c02f32)

Reply:
@aadak99 do you think you can tackle this ?

It's about adding a class "show-for-xlarge-up" for example. See https://get.foundation/sites/docs-v5/components/visibility.html

Reply:
@stephanegigandet  @alexgarel 
I should be able to handle this as it is just an addition of a class. I'll have it done by tonight EST!